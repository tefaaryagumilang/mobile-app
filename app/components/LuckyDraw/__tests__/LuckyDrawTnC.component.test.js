import React from 'react';
import renderer from 'react-test-renderer';

import LuckyDrawTnC from '../LuckyDrawTnC.component';

jest.mock('../LuckyDrawTnC.component');
describe('LuckyDrawTnC component', () => {
  it('LuckyDrawTnC: renders correctly', () => {
    const tree = renderer.create(<LuckyDrawTnC />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
