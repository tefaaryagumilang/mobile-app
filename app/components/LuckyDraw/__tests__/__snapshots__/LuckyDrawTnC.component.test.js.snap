// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LuckyDrawTnC component LuckyDrawTnC: renders correctly 1`] = `null`;
